Styles.css

```diff
@@ -713,3 +713,469 @@ body {
   margin-bottom: 8px;
```

Comment:
**[Optional]**

- [ ] Your PR summary is descriptive but please add it in more detail so the viewer would be able to see how you are making your design :blush: